### PR TITLE
Link to new Discord

### PR DESCRIPTION
**Pull request by nickmccurdy**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -429,7 +429,7 @@ For questions related to using the library, please visit a support community
 instead of filing an issue on GitHub.
 
 - [Spectrum][spectrum]
-- [Reactiflux on Discord][reactiflux]
+- [Discord][discord]
```
Comment (kentcdodds) [inline README.md:432]: Let's ditch the spectrum link too

Comment (nickmccurdy) [reply README.md:431]: Done, what about the badge though?

File: README.md [hunk 2]
```diff
@@ -428,8 +428,7 @@ a 👍. This helps maintainers prioritize what to work on.
 For questions related to using the library, please visit a support community
 instead of filing an issue on GitHub.
```
Comment (afontcu) [inline README.md:429]: (Unrelated, but I think we're missing an "l" here)

```suggestion
instead of filling an issue on GitHub.
```

Comment (nickmccurdy) [reply README.md:429]: I think the usage here is intentionally "filing" as in "to file paperwork". "filling" would technically also work in this context but I feel like it has different semantics, it's more about completing an issue than submitting it.

Comment (afontcu) [reply README.md:429]: oh, right, that makes sense. thanks!

File: README.md (hunk 1 above)
Comment (kentcdodds) [reply README.md:431]: Let's get rid of all spectrum badges. Is there a discord badge we can use? I've seen some that show how many are online

Comment (nickmccurdy) [reply README.md:431]: What do you prefer?

[Discord](https://discord.gg/c6JN9fM)
[Discord](https://discord.gg/c6JN9fM)

You can also customize further by clicking on the Discord badge here https://shields.io/category/chat

Comment (kentcdodds) [reply README.md:431]: Let's do the second one. Looks cooler.

**Comment by kentcdodds** [comment]
Thank you! Let's do this in all the repos! 👍👍

**Comment by nickmccurdy** [comment]
https://github.com/orgs/testing-library/projects/1#card-40465069

**Comment by afontcu** [comment]
on it! :) https://github.com/testing-library/vue-testing-library/pull/148

**Pull request merged**